haigha/transports/socket_transport.py

```diff
@@ -29,26 +34,33 @@ def connect(self, (host, port), klass=socket.socket):
         Connect assuming a host and port tuple.
         '''
```

Comment:
I don't have a preference and never attempted to correctly document all the methods in this project. I see sphinx more often than not and that's the only reason I'd suggest it.